build/helper/metadata_add_all.py

```diff
@@ -149,12 +149,15 @@ def _add_default_value_name_for_docs(parameter, module_name):
 _repeated_capability_parameter_names = ['channelName', 'channelList', 'channel', 'channelNameList']
 
 
-def _add_method_template_filename(f):
-    '''Adds 'method_template_filename' value to function metadata if not found. This is the mako template that will be used to render the method.'''
-    if 'method_template_filename' not in f:
-        f['method_template_filename'] = 'session_default_method.py.mako'
-    if f['method_template_filename'][0] != '/':
-        f['method_template_filename'] = '/' + f['method_template_filename']
+def _add_method_template_filenames(f):
+    '''Adds a list of 'method_template_filenames' value to function metadata if not found. This are the mako templates that will be used to render the method.'''
+    if 'method_template_filenames' not in f:
+        f['method_template_filenames'] = ['session_default_method.py.mako']
+    # Prefix the templates with a / so mako can find them. Not sure mako it works this way.
+    prefixed_filenames = []
+    for filename in f['method_template_filenames']:
+        prefixed_filenames.append('/' + filename if filename != '/' else filename)
```

Comment:
Shouldn't the statement in the append be `'/' + filename if filename[0] != '/' else filename`

Note the `[0]`

Reply:
fixed